test/parallel/test-fs-open-flags.js

```diff
@@ -65,6 +66,12 @@ assert.strictEqual(stringToFlags('xa+'), O_APPEND | O_CREAT | O_RDWR | O_EXCL);
 assert.strictEqual(stringToFlags('as+'), O_APPEND | O_CREAT | O_RDWR | O_SYNC);
 assert.strictEqual(stringToFlags('sa+'), O_APPEND | O_CREAT | O_RDWR | O_SYNC);
 
+assert.strictEqual(stringToFlags('mr'), UV_FS_O_FILEMAP | O_RDONLY);
+assert.strictEqual(stringToFlags('mw'),
+                   UV_FS_O_FILEMAP | O_TRUNC | O_CREAT | O_WRONLY);
+assert.strictEqual(stringToFlags('max+'),
+                   UV_FS_O_FILEMAP | O_APPEND | O_CREAT | O_RDWR | O_EXCL);
+
 ('+ +a +r +w rw wa war raw r++ a++ w++ x +x x+ rx rx+ wxx wax xwx xxx')
```

Comment:
You could add a `common.expectsError()` to this test that checks `'m' + flags` is an error.